lib/stm32/f4/pwr.c

```diff
@@ -39,10 +39,9 @@
 void pwr_set_vos_scale(enum pwr_vos_scale scale)
 {
 	/*
-	 * This function only works fully for F405xx/07xx and F415xx/17xx.
-	 * Scale 1 mode is power saving and is the default at reset.
-	 * Scale 2 mode is high performance.
-	 * TODO: add support for Scale 3 mode
+	 * Scale 1 mode: max 168MHz (high performance, default at reset).
+	 * Scale 2 mode: max 144MHz (power saving).
+	 * Scale 3 mode: max 120MHz (STM32F42xxx/3xxx only).
```

Comment:
I tried to squash the two commits earlier but it seems not to have worked. I think I fixed it now. As for Mode 3 support, I know how to go about adding it, but I don't have F42xxx hardware to test it on.